packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -302,3 +302,16 @@ export const isTestProfileStableFlavor = (): boolean => {
     throw 'Faled to find Communication React Flavor env variable';
   }
 };
+
+/**
+ * Helper function to create query arg to store model data for Fake Chat Adapter
+ */
+export const stringifyChatModel = (users: { displayName: string }[]): string => {
+  return JSON.stringify({
```

Comment:
doing multiple json stringifies can get extremely complicated. Can we chat later about the final format that we want for our string of chat users? 

It can become a serialization/deserialization nightmare

Reply:
Removed this helper function because JSON.stringify for serialization and JSON.parse for deserialization works. Thanks.